src/routes/transactions/__tests__/controllers/list-multisig-transactions-by-safe.transactions.controller.spec.ts

```diff
@@ -154,7 +154,7 @@ describe('List multisig transactions by Safe - Transactions Controller (Unit)',
       const getMultisigTransactionsUrl = `${chain.transactionService}/api/v1/safes/${safeAddress}/multisig-transactions/`;
       const getSafeUrl = `${chain.transactionService}/api/v1/safes/${safeAddress}`;
       const getContractUrlPattern = `${chain.transactionService}/api/v1/contracts/`;
-      const getTokenUrlPattern = `${chain.transactionService}/api/v1/tokens/`;
+      const getTokenUrlPattern = `${chain.transactionService}/api/v1/tokens/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48`;
```

Comment:
This is totally out of the scope of this PR, but at some point I think we should replace all these JSON payloads coming from files by builder-generated payloads, what do you think? We could create an issue for it.

Reply:
Yeah I agree 👍 I think it should be tackled in a different PR as you mentioned. Created an issue: https://github.com/safe-global/safe-client-gateway-nest/issues/386